### ISSUE TITLE
SPPermissionsListHeaderView class is missing layoutMarginsGuide.bottomAnchor

### ISSUE THREAD

**Issue by alzin** [ivanvorobei/SPPermissions#231]
I just have tried the libraries and I was able to install it via CocoaPods but when I compile once after I opened the Xcode workspace again I got an error in the SPPermissionsListHeaderView because **bottomAnchor** misses the **layoutMarginsGuide**, so once I edited that the I built the program successfully!

```
let bottomAnchor = titleLabel.bottomAnchor.constraint(equalTo: layoutMarginsGuide.bottomAnchor, constant: -25)
        bottomAnchor.priority = .init(900)
        bottomAnchor.isActive = true
```

**Comment by ivanvorobei** [comment]
Can you provide error which you got?

**Comment by alzin** [comment]
@ivanvorobei 

After inserting these lines in Podfile and install them,
>pod 'SPPermissions/Bluetooth'
 > pod 'SPPermissions/LocationWhenInUse'
 > pod 'SPPermissions/LocationAlways'

I build the project and get the following error as the screenshot shows!
<img width="1440" alt="Screen Shot 2021-05-18 at 4 32 47 PM" src="https://user-images.githubusercontent.com/35889862/118611066-6852e400-b7f7-11eb-97e1-59af30dcc038.png">
To solve it just added layoutMarginsGuide like this!
<img width="1440" alt="Screen Shot 2021-05-18 at 4 39 29 PM" src="https://user-images.githubusercontent.com/35889862/118611287-a05a2700-b7f7-11eb-9102-38eab173f5fb.png">

**Comment by ivanvorobei** [comment]
Its look strange. In this case no need attach to layout margins. I release new version in 5 minutes.

**Comment by ivanvorobei** [comment]
Thanks for full info.

**Comment by ivanvorobei** [comment]
Released `6.0.5` version, please, update.
